lib/amplitude_api/event.rb

```diff
@@ -81,11 +86,17 @@ def to_hash
       serialized_event[:user_id] = user_id
       serialized_event[:event_properties] = event_properties
       serialized_event[:user_properties] = user_properties
+      serialized_event = add_optional_properties(serialized_event)
```

Comment:
Ah got it, I was reading the diff incorrectly. `add_optional_properties` didn't use to exist! 👍